publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java

```diff
@@ -98,7 +107,11 @@ public void setUp() throws NotFoundException {
         prepareIdentityServiceClient();
         publicationService = new ResourceService(client, Clock.systemDefaultZone());
         ticketService = new TicketService(client);
-        handler = new DeletePublicationHandler(publicationService, ticketService, environment, identityServiceClient);
+        eventBridgeClient = mock(EventBridgeClient.class);
```

Comment:
We have test implementation of EventBridgeClient.

`eventBridgeClient = new FakeEventBridgeClient(ApplicationConstants.EVENT_BUS_NAME);
`
You can than use something like this to verify that event has been persisted:

```
return eventBridgeClient.getRequestEntries()
                   .stream()
                   .map(PutEventsRequestEntry::detail)
                   .map(event -> 
attempt(() -> JsonUtils.dtoObjectMapper.readValue(event, YourClass.class)).orElseThrow())
                   .collect(Collectors.toList());
```